src/sentry/static/sentry/app/views/performance/vitals-cards.tsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {Location} from 'history';
+
+import space from 'app/styles/space';
+import EventView from 'app/utils/discover/eventView';
+import {Organization} from 'app/types';
+import {Panel} from 'app/components/panels';
+import {formatPercentage} from 'app/utils/formatters';
+import VitalsCardsDiscoverQuery from 'app/views/performance/vitalDetail/vitalsCardsDiscoverQuery';
+import {getAggregateAlias, WebVital} from 'app/utils/discover/fields';
+import Link from 'app/components/links/link';
+import {t} from 'app/locale';
+
+import {
+  vitalAbbreviations,
+  vitalDetailRouteWithQuery,
+  vitalsBaseFields,
+  vitalsThresholdFields,
+} from './vitalDetail/utils';
+
+type Props = {
+  eventView: EventView;
+  organization: Organization;
+  location: Location;
+};
+
+export default function VitalsCards(props: Props) {
+  const {eventView, organization, location} = props;
+  const vitalsView = eventView.clone();
+
+  const shownVitals = [
+    WebVital.FP,
+    WebVital.FCP,
+    WebVital.LCP,
+    WebVital.FID,
+    WebVital.CLS,
+  ];
+
+  return (
+    <VitalsCardsDiscoverQuery
+      eventView={vitalsView}
+      orgSlug={organization.slug}
+      location={location}
+    >
+      {({isLoading, tableData}) => (
```

Comment:
Should this handle errors as well?

Reply:
I thought about it and wasn't sure about those cards setting an error, I'd prefer if they just displayed (`-`), not sure though